build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
```

Comment:
If `moment()` is constructed with a non-ISO formatted date string like `April 27, 2015`, it will log a warning:

```
Deprecation warning: moment construction falls back to js Date. This is discouraged and will be removed in upcoming major release. Please refer to https://github.com/moment/moment/issues/1407 for more info.
```

The way around this is to pass in `Date` instances instead, if non-ISO formats are to be supported. If the deprecation is something we wish to ignore, then we can unwrap the values and pass them as strings directly to `moment()`.